### DISCUSSION TITLE
Breaking Changes in v1.1.2

### DISCUSSION THREAD

**Discussion by StokleyO** [joshnuss/svelte-stripe#94] [Q&A] [answered]
Hey everyone,

I'm still on svelte-stripe v0.0.22 -- are there any breaking changes if I were to upgrade to v1.1.2?

**Comment by joshnuss** [comment, marked as answer]
Hi @StokleyO,

There should be no breaking changes. Mostly just addition of new components, and adding some missing events and attributes.

I started using [`semantic-release`](https://github.com/semantic-release/semantic-release) for auto-deploying the NPM package, hence the big jump in version numbers.

**Reply by StokleyO** [reply]
Perfect!! Thanks for the speedy response @joshnuss, btw, if you need some help maintaining/contributing to svelte-stripe, I would love to help out in any way I can!!

**Reply by joshnuss** [reply]
Thanks @StokleyO!

PRs and ideas are always welcome, I probably should publish a roadmap
